src/schema/2.0/w3c.ts

```diff
@@ -0,0 +1,52 @@
+import { getData, SignedDocument, v2 } from "../..";
+
+interface Salt {
+  type: string;
+  value: string;
+  path: string;
+}
+
+const deepMap = (value: any, path: string): Salt[] => {
+  if (Array.isArray(value)) {
+    return value.flatMap((v, index) => deepMap(v, `${path}[${index}]`));
+  }
+  if (typeof value === "object") {
+    return Object.keys(value).flatMap(key => deepMap(value[key], path ? `${path}.${key}` : key));
+  }
+  if (typeof value === "string") {
+    const [salt, type] = value.split(":");
+    return [{ type, value: salt, path }];
+  }
+  throw new Error(`unexpected element  ${value} => ${path}`);
+};
+const getSalts = (document: SignedDocument<v2.OpenAttestationDocument>): Salt[] => {
+  return deepMap(document.data, "");
+};
+
+/**
+ * IT S A FUCKING POC GOT IT ?
+ * This function is not production ready and is a simple POC to demonstrate that we are able to transform our document to W3C VC
+ * https://www.w3.org/TR/vc-data-model/#types
+ */
+export const mapToW3cVc = (document: SignedDocument<v2.OpenAttestationDocument>): any => {
```

Comment:
Should we expose this in the module? maybe something with a experimental label? It does not seemed like this function   is used anywhere else other than the test.

Reply:
I will export and rename with a weird prefix like `
`__EXPERIMENTAL__DO_NOT_USE_IN_PROD`

Reply:
done => `__unsafe__mapToW3cVc`